.github/workflows/ci-test.yml

```diff
@@ -4,28 +4,24 @@ jobs:
   test:
     name: (JDK ${{ matrix.jdk }} / ${{ matrix.os }}) Test
     env:
-      MAVEN_OPTS: -XX:+IgnoreUnrecognizedVMOptions --illegal-access=debug -XX:StartFlightRecording=maxsize=5g,disk=true,dumponexit=true,settings=default,filename=./
+      MAVEN_OPTS: --illegal-access=debug -XX:StartFlightRecording=maxsize=5g,disk=true,dumponexit=true,settings=default,filename=./
```

Comment:
You can even drop `--illegal-access=debug` as this is no longer supported from Java 17+

Reply:
@reinhapa Thanks, I have pushed an additional commit to address that.